server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -633,7 +679,7 @@ void removeAndDeletePoolsOnOtherOwners(List<Pool> existingPools, Pool pool) {
          * send out the events. Create an event for each pool that could change,
          * even if we won't use them all.
          */
-        if (existingPools == null || existingPools.isEmpty()) {
+        if (CollectionUtils.isEmpty(existingPools)) {
```

Comment:
Why are we abstracting out a null check? This is slower, and forces a dependency on that utility.